Remotion/Web/Core/Res/HTML/SmartPage.js

```diff
@@ -1176,7 +1176,7 @@ function SmartPage_Context(
     };
     if (_submitState == null)
       _submitState = submitState;
-    else if (isAutoPostback || submitterElement == null || submitterElement.id !== _submitState.Submitter.id)
+    else if (_submitState.IsAutoPostback && !isAutoPostback)

```

Comment:
Bump for continuity: when filling in a textbox, then hitting the async link and then hitting the async button, it does a postback for the textbox and the button, but not for the link (which i clicked before the button). Please report back to Michael to check if that behavior is intended.